src/Redmine/Api/Group.php

```diff
@@ -71,7 +74,7 @@ public function create(array $params = [])
         if (
             !isset($params['name'])
         ) {
-            throw new \Exception('Missing mandatory parameters');
+            throw new MissingParameterException('Theses parameters are mandatory: `name`');
```

Comment:
```suggestion
            throw new MissingParameterException('Missing mandatory parameters `name`');
```
?

Reply:
A `MissingParameterException` is thrown if one or more mandatory parameters are missing. I tried to find a generic message, that fits in all places but without the need of logic code to name exactly the missing parameters.